### PR TITLE
Updated Changelog for 12.3

### PR DESCRIPTION
**Pull request by dobooth**
## Description

This PR updates Changelog.md with PWA-Studio 12.3.0 release information.

## Related Issue

https://jira.corp.magento.com/browse/PWA-2611

## Verification Steps

Verify all letters, numbers, and spaces are in the correct order.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,108 +1,91 @@
...
--  Safari (macOS version) does not show toast messages or indicators when Venia switches between online and offline. This is an issue with Safari, not Venia. Safari always reports `true` for `navigator.onLine` — even when offline. We have submitted this issue to Apple. If you have an Apple account, you can search for the issue using this Feedback ID: FB9802994.
+-  None!
 
 ## Upgrading from a previous version
 
-Use the steps outlined in this section to update your [scaffolded project][] from 12.1.0 to 12.2.0.
+Use the steps outlined in this section to update your [scaffolded project][] from 12.2.0 to 12.3.0.
 See [Upgrading versions][] for more information about upgrading between PWA Studio versions.
 
 [scaffolded project]: https://magento.github.io/pwa-studio/tutorials/pwa-studio-fundamentals/project-setup/
```
Comment (jcalcaben) [inline CHANGELOG.md:91]: Should update these links to point to the new site

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,108 +1,91 @@
...
+| Bug   |  PWA multiple wish lists: storefront still showing create wishlist button when reaching the max number of allowed wish lists configured in Commerce    | [3668][]                      |
+| Bug   | Header - Logo dimensions overlap with main content                                                                                                | [3659][]                      |
+| Bug   | Click target for long-spanning tabs is inconsistent                                                                                          | [3676][]                      |
+| Bug   | Venia sample data personalized content not updating                                                                                               | [20][]                        |
 
 ## Documentation updates
 
--  **contactUs Mutation**: Added the [contactUs][] mutation topic to the GraphQL Developer Guide.
+-   Added [best practices documentation](https://developer.adobe.com/dreamweaver/guides/best-practices/index.html) for Customer Segments.
```
Comment (jcalcaben) [inline CHANGELOG.md:80]: Broken link

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,108 +1,91 @@
...
-| Bug   | Fix console warning for currency                                                                                                        | [3622][]          |
-| Bug   | Remove server chunk generation from webpack                                                                                             | [3608][]          |
-| Bug   | \[Issue] Improve Gallery/AddToCartButton.js                                                                                             | [3573][]          |
-| Bug   | \[bug]: Breadcrumbs 'Shop' link redirects to '/shopnull'                                                                                | [3603][]          |
-| Bug   | Newsletter form displays two errors                                                                                                     | [3584][]          |
-| Bug   | Home page tabbing leads to page transition                                                                                              | [3625][]          |
-| Bug   | Browser Reload on any Category load all products from default category                                                                  | [3655][], [18][]  |
+
+## Highlights
```
Comment (jcalcaben) [inline CHANGELOG.md:8]: Suggest grouping this list of highlights into the high level epics they are associated with:

### Personalized Content Support



* [3713][]
* [3][]
* [19][] [3670][]
* [3609][]
* [2][] [3587][]
* Docs related to this epic

### PageBuilder Content optimization



* [3711][]
* [3712][]
* [16][] [17][]

### ReCaptcha support



* [34][] [3691][]
* [3702][]
* [20][] [3604][]

**Review by jcalcaben — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (dobooth) [reply CHANGELOG.md:91]: Good point. I thought I updated those numbers.

File: CHANGELOG.md (hunk 2 above)
Comment (dobooth) [reply CHANGELOG.md:80]: It will resolve when we push the 12.3 work.

Comment (jcalcaben) [reply CHANGELOG.md:80]: PWA Studio best practices guide will be under the `dreamweaver` namespace?

File: CHANGELOG.md (hunk 3 above)
Comment (dobooth) [reply CHANGELOG.md:7]: So helpful

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,228 +1,178 @@
...
-[3664]: https://github.com/magento/pwa-studio/pull/3664
-[3665]: https://github.com/magento/pwa-studio/pull/3665
+| `venia-concept`                     | **12.3.0**     |
+| `venia-ui`                          | **9.3.0**      |
+| `magento2-pwa`                      | **0.2.0**      |
+| `magento2-pwa-commerce`             | **0.0.1**      |
+| `magento-venia-sample-data-modules` | **0.0.2**      |
+| `magento-venia-sample-data-modules-ee`| **0.0.1**    |
+| `magento2-upward-connector`         | **2.0.1**      |
+| `upward-php`                        | **2.0.1**      |
```
Comment (dpatil-magento) [inline CHANGELOG.md:128]: @dobooth I updated test plan. no change for upward repo's.
magento2-upward-connector >> 2.0.0 
upward-php >> 2.0.0

File: CHANGELOG.md (hunk 2 above)
Comment (dobooth) [reply CHANGELOG.md]: It was, back in 2004. Fixed.

**Review by dpatil-magento — APPROVED** [review]

**Review by dpatil-magento — APPROVED** [review]

**Comment by bdenham** [comment]
One note about the Changelog: that [files][] link needs a real PR number from GitHub.

**Review by dpatil-magento — APPROVED** [review]

**Pull request merged**